.travis.yml

```diff
@@ -59,6 +59,8 @@ jobs:
 
     - node_js: "12.16.3"
       name: 'Browser tests: Linux/Chromium'
+      addons:
+        chrome: stable
```

Comment:
needed to ensure Travis has an up to date chrome that web test runner can use

Reply:
We could also point it to the bundled chromium which puppeteer already brings. Let me know if I can help here :)

Reply:
@LarsDenBakker thanks! I'm quite keen to see if it's stable on a regular Chrome, but if not we'll reach for that and I expect I'll ping you for help :D